static/js/publisher/pages/RegisterSnap/RegisterSnap.tsx

```diff
@@ -0,0 +1,108 @@
+import { ReactNode, useState } from "react";
+import { Link } from "react-router-dom";
+import { Notification } from "@canonical/react-components";
+
+import RegisterSnapForm from "./RegisterSnapForm";
+
+import { filterAvailableStores } from "../../utils";
+
+import { useBrandStores } from "../../hooks";
+
+import type { RegistrationResponse } from "./local-types";
+
+function RegisterSnap(): ReactNode {
+  const [isSending, setIsSending] = useState<boolean>(false);
+  const [registrationResponse, setRegistrationResponse] =
+    useState<RegistrationResponse>();
+  const { data, isLoading } = useBrandStores();
+
+  const availableStores = filterAvailableStores(data || []);
+
+  return (
+    <div className="u-fixed-width">
+      {registrationResponse?.error_code === "already_registered" ||
+      registrationResponse?.error_code === "already_claimed" ? (
+        <>
+          <h1 className="p-heading--2">
+            <strong>{registrationResponse.snap_name}</strong> is already taken
```

Comment:
Is there a reason to put error message in page title?

Feels a bit odd to change page title from "Register snap" based on error response. And a bit redundant, as the error notification already shows all the information?

Reply:
Even more so - this only happens for these 2 error types, other types don't change the title just show notification (code below).